Source/Examples/WPF/WpfExamples/WpfExamples.csproj

```diff
@@ -73,6 +73,9 @@
     <Compile Include="Examples\BindingObservableCollectionDemo\MainWindow.xaml.cs">
       <DependentUpon>MainWindow.xaml</DependentUpon>
     </Compile>
+    <Compile Include="Examples\BoxPlotSeriesDemo\MainWindow.xaml.cs">
```

Comment:
Oh I didn't see the NET 4.0 project file as it isn't loaded by default in VS. I will also update it.